### DISCUSSION TITLE
Issue about demo_cam_reader.html

### DISCUSSION THREAD

**Discussion by 549393092** [zxing-cpp/zxing-cpp#771] [General]
Dear,

Could your please check the demo code
https://github.com/zxing-cpp/zxing-cpp/blob/master/wrappers/wasm/demo_cam_reader.html#L135
which will receive parameter "front" and "back" to trigger camera.
However, it seems not work. Instead it works with "user" and "environment". see [MDN doc](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia).

Best Regards

**Comment by axxel** [comment]
Continue discussion in #772
